examples/ultrahdr_app.cpp

```diff
@@ -298,6 +296,10 @@ class UltraHdrAppInput {
         mQuality(95),
         mOTf(oTf),
         mOfmt(oFmt),
+        mFullRange(UHDR_CR_UNSPECIFIED),
+        mMapDimensionScaleFactor(1),
```

Comment:
Same comment as I mentioned in the other CL

Reply:
Done

Reply:
Done